pom.xml

```diff
@@ -34,6 +34,12 @@
         <relativePath />
     </parent>
 
+    <properties>
+        <!-- https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-release.html -->
+        <!-- Overwrites the release property (jdk 9+) for maven-compiler-plugin defined in the securebanking-parent -->
+        <java.version>11</java.version>
```

Comment:
The link states that you should set: `maven.compiler.release` property? Is `java.version` correct?

Reply:
Nevermind, I've looked at the parent pom!

Reply:
This the parameter value to set the release in maven compiler plugin managed by securebanking-parent:
```xml
<pluginManagement>
            <plugins>
                <plugin>
                    <groupId>org.apache.maven.plugins</groupId>
                    <artifactId>maven-compiler-plugin</artifactId>
                    <version>${maven-compiler-plugin.version}</version>
                    <configuration>
                        <release>${java.version}</release>
                        <verbose>false</verbose>
                        <fork>true</fork>
                    </configuration>
                </plugin>
```